modules/UKBiobank.py

```diff
@@ -0,0 +1,201 @@
+from settings import Config
+from common.HGNCParser import GeneParser
+import sys
+import logging
+import datetime
+import opentargets.model.core as opentargets
+import opentargets.model.bioentity as bioentity
+import opentargets.model.evidence.core as evidence_core
+import opentargets.model.evidence.genetics as evidence_genetics
+import opentargets.model.evidence.linkout as evidence_linkout
+import opentargets.model.evidence.association_score as association_score
+
+__copyright__ = "Copyright 2014-2018, Open Targets"
+__credits__   = ["ChuangKee Ong"]
+__license__   = "Apache 2.0"
+__version__   = "1.2.8"
+__maintainer__= "ChuangKee Ong"
+__email__     = ["data@opentargets.org"]
+__status__    = "Production"
+
+class UKBiobank():
```

Comment:
as python 3 you dont need to put that just `class UKBiobank:` otherwhise python2 `class UKBiobank(object):` 
https://stackoverflow.com/questions/4906014/python-default-inheritance

Reply:
what I want to say is `()` are unnecesary